staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

```diff
@@ -67,9 +70,10 @@ var (
 type ExplainOptions struct {
 	genericiooptions.IOStreams
 
-	CmdParent  string
-	APIVersion string
-	Recursive  bool
+	CmdParent        string
+	APIVersion       string
+	Recursive        bool
+	VerboseRecursive bool
```

Comment:
I'd prefer if possible for this to be refactored so that the new flag introduces an orthogonal concept. Right now there is overlap with `recursive`. For example for verbose recursive we might want a `Long-form` flag to be set and the `Recursive` flag.

Reply:
Thanks @alexzielenski , if i understand it well we should use something like this `kubectl explain pods --recursive --long-form`. I am not sure what should i print when user does this: `kubectl explain pods --long-form=false`.